addons/backpacks/mr3000/ui/mr3000.hpp

```diff
@@ -289,12 +271,7 @@ class additional: HiddenRotator
     w = 0.0367463 * safezoneW;
     h = 0.0812181 * safezoneH;
     tooltip = ECSTRING(core,set_additional);
-    action = QUOTE(
-        playSound 'TFAR_rotatorPush';
-        [ARR_2(TF_lr_dialog_radio,TF_lr_dialog_radio call TFAR_fnc_getLrChannel)] call TFAR_fnc_setAdditionalLrChannel;
-        call TFAR_fnc_updateLRDialogToChannel;
-        [ARR_2(TF_lr_dialog_radio, true)] call TFAR_fnc_showRadioInfo;
-    );
+    action = QUOTE(playSound 'TFAR_rotatorPush';[ARR_2(TF_lr_dialog_radio,TF_lr_dialog_radio call TFAR_fnc_getLrChannel)] call TFAR_fnc_setAdditionalLrChannel;call TFAR_fnc_updateLRDialogToChannel;[ARR_2(TF_lr_dialog_radio, true)] call TFAR_fnc_showRadioInfo;);
```

Comment:
Can't this really not be multiline? Multiline should work.

Reply:
Don't know. I've just exhumed out an old branch to speed up the process. I'll try

Reply:
Result?

Reply:
Bump